nvtabular/inference/triton/model_hugectr.py

```diff
@@ -87,7 +87,7 @@ def execute(self, requests: List[InferenceRequest]) -> List[InferenceResponse]:
 
             if "cats" in self.column_types:
                 output_df[self.column_types["cats"]] = (
-                    output_df[self.column_types["cats"]] + self.slot_sizes
+                    output_df[self.column_types["cats"]] + self.offsets
```

Comment:
How about doing this in place? This is more readable and can avoid allocating memory 

```python
output_df[self.column_types["cats"]] += self.offsets
```

Reply:
Sure, we can do that as well. But, I am not sure if cuDF is allocating or not allocation memory space when we do it that way. Something we should test, I guess.